scaleway/helpers_container.go

```diff
@@ -35,3 +35,65 @@ func containerAPIWithRegionAndID(m interface{}, id string) (*container.API, scw.
 	}
 	return api, region, id, nil
 }
+
+func setCreateContainerRequest(d *schema.ResourceData, region scw.Region) (*container.CreateContainerRequest, error) {
+	// required
+	nameRaw := d.Get("name")
+	namespaceID := d.Get("namespace_id")
+
+	name := expandOrGenerateString(nameRaw.(string), "co")
+	privacyType := d.Get("privacy") // default unknown_privacy
```

Comment:
Why do you specify unknown_privacy? It is something that is purely internal and should not be used by client. Default privacy is private and/or public but not unknown.
Protocol default is probably http1 but not unknown.

Reply:
type unknown is no longer considered